jerry-core/ecma/operations/ecma-arraybuffer-object.c

```diff
@@ -319,14 +349,18 @@ ecma_builtin_arraybuffer_slice (ecma_value_t this_arg,
 
   /* 11. */
   ecma_value_t ctor;
+  #if JERRY_BUILTIN_SHAREDARRAYBUFFER
   if (ecma_is_shared_arraybuffer (this_arg))
   {
     ctor = ecma_op_species_constructor (object_p, ECMA_BUILTIN_ID_SHARED_ARRAYBUFFER);
   }
   else
   {
+  #endif
     ctor = ecma_op_species_constructor (object_p, ECMA_BUILTIN_ID_ARRAYBUFFER);
+  #if JERRY_BUILTIN_SHAREDARRAYBUFFER
   }
+  #endif
```

Comment:
ditto, also missing #endif comment

Reply:
Moreover, how about guarding only the builtin_id?

Reply:
Can you please explain what do you mean?